runtime/scripts/question.js

```diff
@@ -1079,7 +1087,7 @@ Question.prototype = /** @lends Numbas.Question.prototype */
             this.store.answerRevealed(this);
         }
         this.exam && this.exam.updateScore();
-        this.signals.trigger('revealed');
+        this.events.trigger('revealAnswer');
```

Comment:
no, that signal should stay.

Reply:
that signal is used by many questions that are in use, so this would be a breaking change if it was merged in